.ci/spec/run-test.sh

```diff
@@ -4,31 +4,27 @@ csource="${BASH_SOURCE[0]}"
 while [ -h "$csource" ] ; do csource="$(readlink "$csource")"; done
 root="$( cd -P "$( dirname "$csource" )/../../" && pwd )"
 
-. "${root}/.ci/spec/env.sh"
 . "${root}/.ci/common.sh"
+. "${root}/.ci/spec/env.sh"
 
 pushd "${root}" > "${output}"
 
 # Assumes packet is already set up
-
-# job_id is provided by common.sh
-export PACKET_EXEC_REMOTE_DIRECTORY="${job_id}"
+unset PACKET_EXEC_PRE_BUILTINS
```

Comment:
When this is set, it re-installs all the vmware tools and causes `vagrant --provision` to fail

Reply:
:+1: though we should have a look at that helper and see why it's not a no-op at this point.